RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -312,62 +295,79 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 			costMajor = mp_cost($skill[Stream of Sauce]);
 		}
 
-		if(canUse($skill[Soul Bubble], false) && my_soulsauce() >= 5)
-		{
-			stunner = useSkill($skill[Soul Bubble]);
-			costStunner = mp_cost($skill[Soul Bubble]);
-		}
-
 		if(!contains_text(combatState, "delaymortarshell") && canSurvive(2.0) && haveUsed($skill[Stuffed Mortar Shell]) && canUse($skill[Salsaball], false))
 		{
 			set_property("auto_combatHandler", combatState + "(delaymortarshell)");
 			return useSkill($skill[Salsaball], false);
 		}
-
 		break;
 
 	case $class[Avatar of Boris]:
+		// Boris will have problems if Trusty isn't wielded
+		if (equipped_item($slot[Weapon]) != $item[none])
```

Comment:
1. boris can use the skills when unarmed. and it is certainly better than doing an unarmed attack
2. rather than wrapping the whole function this should be an interrupt style if "if X then return/abort/etc"
3. boris should abort if it somehow goes into combat without trusty. we specifically have code to try to force it to always wear trust and it should not ever unequip it. if it is unequipping it something has gone terribly wrong and we should really abort.